action.yml

```diff
@@ -88,7 +88,7 @@ runs:
   using: composite
   steps:
     - name: Checkout
-      uses: docker://public.ecr.aws/l7o7z1g8/actions/cloudbees-io-checkout:v1.0.7
+      uses: docker://registry.saas-dev.beescloud.com/staging/cloudbees-io-checkout:0.0.58
```

Comment:
Once this PR is merged to `main`, the image version will be updated to the corresponding `main` branch version defined in the workflow, which is for this ticket `v1.0.8`